src/openms/source/FORMAT/HANDLERS/XQuestResultXMLHandler.cpp

```diff
@@ -63,35 +65,48 @@ namespace OpenMS
     std::map< String, UInt> XQuestResultXMLHandler::months = boost::assign::map_list_of("Jan", 1)
```

Comment:
should now work without boost
https://stackoverflow.com/questions/3250123/what-would-a-stdmap-extended-initializer-list-look-like